crml/cennzx/src/benchmarking.rs

```diff
@@ -0,0 +1,160 @@
+/* Copyright 2019-2020 Centrality Investments Limited
+*
+* Licensed under the LGPL, Version 3.0 (the "License");
+* you may not use this file except in compliance with the License.
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+* You may obtain a copy of the License at the root of this project source code,
+* or at:
+*     https://centrality.ai/licenses/gplv3.txt
+*     https://centrality.ai/licenses/lgplv3.txt
+*/
+
+//! Attestation benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use frame_benchmarking::{account, benchmarks, whitelisted_caller};
+use frame_system::RawOrigin;
+
+use crate::Module as Cennzx;
+
+benchmarks! {
+	_{ }
+
+	buy_asset {
+		let investor: T::AccountId = whitelisted_caller();
+		let buyer: T::AccountId = account("buyer", 0, 0);
+
+		let core_asset_id = <Cennzx<T>>::core_asset_id();
+		let asset_a: T::AssetId = 2u32.into();
```

Comment:
nit: is it possible to use const for the asset IDs?

Reply:
Used const